applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -20,20 +20,32 @@ LOG_MODULE_REGISTER(sw_codec_select, CONFIG_SW_CODEC_SELECT_LOG_LEVEL);
 
 static struct sw_codec_config m_config;
 
+#ifdef CONFIG_AUDIO_SAMPLE_RATE_CONVERSION
```

Comment:
We briefly discussed this. THere should only be need for a single CONFIG which is if the sample rate library has been compiled in or not.